jetbrains-core/src/software/aws/toolkits/jetbrains/services/cloudwatch/logs/editor/CloudWatchLogGroup.kt

```diff
@@ -154,9 +155,7 @@ class CloudWatchLogGroup(
 
     private fun populateModel() = try {
         val streams = client.describeLogStreamsPaginator(DescribeLogStreamsRequest.builder().logGroupName(logGroup).build())
-        streams.filterNotNull().firstOrNull()?.logStreams()?.let {
-            tableModel.items = it
-        }
+        tableModel.items = streams.asSequence().map { it.logStreams() }.toList().flatten()
```

Comment:
```suggestion
         tableModel.items = streams.asSequence().flatMap { it.logStreams().asSequence() }.toList()
```

Reply:
Committed the suggestion (with proper formatting for ktlint)